trailing_spaces.py

```diff
@@ -427,6 +482,33 @@ def on_pre_save(self, view):
         if ts_settings.get("trailing_spaces_trim_on_save"):
             view.run_command("delete_trailing_spaces")
 
+    def on_close(self, view):
+        # untrack
+        active_views.pop(view.id(), None)
+
+    def update_view_on_scroll(self, view):
```

Comment:
Don't like that name as it doesn't run on scroll but rather periodically regardless of scrolling.
Maybe a better option would be `update_if_region_changed`